### DISCUSSION TITLE
doh دقیقا چیه؟

### DISCUSSION THREAD

**Discussion by s-a-moeini** [GFW-knocker/gfw_resist_HTTPS_proxy#29] [Q&A]
سلام ، خسته نباشید ، بنده چون مبتدی هستم در مورد doh متوجه نمیشم چطور باید ازش استفاده کنم ، میشه آموزش و یا توضیح بیشتری بدید که دقیقا به چه کاری میاد و چطور ازش استفاده کنیم

**Comment by GFW-knocker** [comment]
مخفف dns over https هست
برا کدنویس ها خوبه
بزودی میاریمش تو اندروید برا عموم


از chatgpt بپرسید
chatbot.theb.ai
poe.com
